### DISCUSSION TITLE
Future or Updated ARM support for LXC / Proxmox?

### DISCUSSION THREAD

**Discussion by ernenr1** [automatic-ripping-machine/automatic-ripping-machine#746] [General]
Hi there! I'm still new to the ARM project community, and I have been impressed how far the community has gone! I have been running ARM bare metal on Ubuntu 20.04 without any problems for the past few months. However from what I've gathered in recent weeks, I see that the community is heading towards focusing all its effort on dockerizing the install of ARM. The shift is noticable when looking at the manual install instructions, instructing those to use the ARM install docker container. 

With that said, I know I can't run ARM bare metal in the long term. With me running Proxmox and having LXC available to me, has the community thought about adding ARM support to LXC? I know someone has written a guide to get ARM working within LXC / Proxmox within the "Related Projects and HowTos" page from Reddit, but that was written five years ago... 

Anyways very grateful to the community and developers of the project! Thank you very much.

**Comment by 1337-server** [comment]
From my very limited understanding, Proxmox doesnt work well with MakeMKV.

Sadly LXC support is something that i dont think we could add.
We just dont have the number of devs & testers we need to support this. But if you are able to make a new updated guide id love to add it to the wiki so that others can follow if they wish.
